supadantic/clients/base.py

```
@@ -12,7 +12,7 @@ class BaseClientMeta(ABCMeta):
     specific attributes).
     """
 
-    pass
+    __slots__ = ()
```

why do we need it?

---

It was a workaround because `wemake-python-styleguide` thinks empty classes are bad. I will remove it and ignore the error